release.json

```diff
@@ -59,7 +59,7 @@
         "WINDOWS_DDPROCMON_SHASUM": "7c13ba75f7a30704a6a4082e4cffc819c846fd6061c53372c8b9908fee11d621"
     },
     "release-a7": {
-        "INTEGRATIONS_CORE_VERSION": "7.54.0",
+        "INTEGRATIONS_CORE_VERSION": "7.54.1-rc.1",
         "OMNIBUS_SOFTWARE_VERSION": "7.54.0",
         "OMNIBUS_RUBY_VERSION": "7.54.0",
```

Comment:
@chouquette just pushed a new tag for `OMNIBUS_RUBY` you should ship it I guess

Reply:
Thanks for the notice, updated